oci/private/descriptor.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
include a file-level comment about what json document you're producing here? bonus points for link to a spec